repo/packages/R/riak/0/marathon.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "{{riak.app-id}}",
+  "instances": {{riak.instances}},
+  "cpus": {{riak.cpus}},
+  "mem": {{riak.mem}},
+  "ports": [0, 0],
+  "uris": ["{{riak.url}}"],
+  "env": {},
+  "args": [
+      "riak_mesos_framework/framework_linux_amd64"
+      ,"-master={{riak.master}}"
+      ,"-zk={{riak.zk}}"
+      {{#riak.ip}}
+      ,"-ip={{riak.ip}}"
+      {{/riak.ip}}
+      {{#riak.hostname}}
+      ,"-hostname={{riak.hostname}}"
+      {{/riak.hostname}}
+      {{#riak.log}}
+      ,"-log={{riak.log}}"
+      {{/riak.log}}
+      {{#riak.user}}
+      ,"-user={{riak.user}}"
+      {{/riak.user}}
+      {{#riak.framework-name}}
+      ,"-name={{riak.framework-name}}"
+      {{/riak.framework-name}}
```

Comment:
there is no need for a guard here... framework-name is always required for frameworks